docs/command-line-tool.md

```diff
@@ -259,16 +273,17 @@ $ run <title> <text> <options> <long-options>
 ```
 
 The arguments for are:
+
 - `<title>` the title for the user gui dialog
 - `<text>` the text for the user gui dialog
-- `<short-options>` the button return values, slash-delimited, 
```

Comment:
Not sure why these have changed, I'll revert it. /after

Reply:
Nope, this is how it should be, I'll leave it.

Reply:
Was a markdown format in vs code :|